src/openfluid/ui/waresdev/GitUIProxy.cpp

```diff
@@ -346,6 +346,21 @@ bool GitUIProxy::clone(const QString& FromUrl, const QString& ToPath,
 // =====================================================================
 
 
+bool GitUIProxy::checkout(const QString& Path, const QString& BranchName)
+{
+  //TOIMPL test this function?
+  QStringList Args = {"checkout", BranchName, "--progress"};
+  launchLocalCommand(Path, Args);
```

Comment:
Why not ussing return code of the git command to verify that the checkout is successful?
Using the git command output works most of the time, but the return code should be more robust (if usable?)